pkg/apis/authorization/types.go

```diff
@@ -38,6 +38,10 @@ type SubjectAccessReview struct {
 	Status SubjectAccessReviewStatus
 }
 
+// +genclient=true
+// +nonNamespaced=true
+// +noMethods=true
```

Comment:
Sorry, I couldn't find documentation on what `+noMethods` accomplishes. Does it limit the actions generated to "Create"?